extension/js/common/inject.ts

```diff
@@ -54,4 +55,20 @@ export class Injector {
     }
   }
 
+  insertEndSessionBtn = async (acctEmail: string) => {
+    $(this.factory.btnEndPPSession())
+    .insertBefore($('.gb_Zd').children().last())
+    .click(Ui.event.prevent('double', async el => {
+      for(const key of (await Store.keysGet(acctEmail))) {
```

Comment:
I recommend to put this logic into `Store.isPassPhraseCurrentlyInSession`

Reply:
Yes, Agree, then there won't be code duplications.